### PR TITLE
Fix tinyMCE editor buttons not working with subform

### PR DESCRIPTION
**Pull request by mvanvu**
Pull Request for Issue # .

### Summary of Changes

This PR for fix the buttons of TinyMCE working is not correct in the subform field. 

### Testing Instructions

1. Config TinyMCE is default editor
2. Create a new content field with type is subform repeatable and choose an editor child field
3. Edit an article and from the subform field save it with 2 editor.
4. By default, the editor field will not show the buttons so you have to edit the file /plugins/fields/repeatable/repeatable.php at the line 74
repeatable
5. Back to the article edit page
- Add an image into the first editor => it works fine
- Add an image into the second editor => it don't work, the image was still inserted into the first editor and the modal not closed. 

### After PR
It should work.

### CODE REVIEW

File: plugins/editors/tinymce/tinymce.php [hunk 1]
```diff
@@ -161,7 +161,7 @@ public function onDisplay(
 
 		$id            = preg_replace('/(\s|[^A-Za-z0-9_])+/', '_', $id);
 		$nameGroup     = explode('[', preg_replace('/\[\]|\]/', '', $name));
-		$fieldName     = end($nameGroup);
+		$fieldName     = join('_', $nameGroup);
```
Comment (SharkyKZ) [inline plugins/editors/tinymce/tinymce.php:164]:
```suggestion
		$fieldName     = implode('_', $nameGroup);
```

File: media/editors/tinymce/js/tinymce.js [hunk 2]
```diff
@@ -44,7 +44,7 @@
 		 * @since 3.7.0
 		 */
 		setupEditor: function ( element, pluginOptions ) {
-			var name = element ? element.getAttribute('name').replace(/\[\]|\]/g, '').split('[').pop() : 'default', // Get Editor name
+			var name = element ? element.getAttribute('name').replace(/\[\]|\]/g, '').split('[').join('_') : 'default', // Get Editor name
```
Comment (Fedik) [inline media/editors/tinymce/js/tinymce.js:47]: why do you change this?

File: media/system/js/subform-repeatable-uncompressed.js [hunk 3]
```diff
@@ -132,6 +132,20 @@
...
+				var eid = this.id;
+				if (Joomla.editors.instances[eid]) {
+					$(this).siblings('.toggle-editor').find('a.btn').unbind('click').on('click', function () {
+						tinyMCE.execCommand('mceToggleEditor', false, eid);
+						return false;
+					});
+				}
+			});
+		}
+
```
Comment (Fedik) [inline media/system/js/subform-repeatable-uncompressed.js]: Sorry, the fix are wrong, even it if it work, it hacky and will fail on any markup changes

Additionally, any such code related to tinymce editor should be placed in `media/editors/tinymce/js/tinymce.js`

Comment (mvanvu) [reply media/system/js/subform-repeatable-uncompressed.js]: Thank you, your're right. I will move it to the tinymce.js

**Pull request closed without merging**